docs/src/main/sphinx/sql/select.md

```diff
@@ -69,6 +69,30 @@ ROLLUP ( column [, ...] )
 
 Retrieve rows from zero or more tables.
 
+## WITH SESSION clause
+
+The `WITH SESSION` clause allows you to [set session and catalog session
+property values](/sql/set-session) applicable for the processing of the current
+select query only. The defined values override any other configuration and
+session property settings. Multiple properties are separated by commas.
+
+The following example overrides the global configuration property
+`query.max-execution-time` with the session property `query_max_execution_time`
+to reduce the time to `2h`. It also overrides the catalog property
+`iceberg.query-partition-filter-required` from the `example` catalog using
+[](/connector/iceberg) setting the catalog session property
+`query_partition_filter_required` to `true`:
+
+```sql
+WITH 
+  SESSION
+    query_max_execution_time='2h',
+    example.query_partition_filter_required='true'
```

Comment:
I believe this query fails unless removing `'`:
```suggestion
    example.query_partition_filter_required=true
```

Reply:
haha .. I had a suspicion my guess was wrong.. thanks for confirming @ebyhr